Dockerfile-periodic

```diff
@@ -11,13 +11,6 @@ RUN apt-get update -yqq \
     locales \
   && apt-get -q clean
 
-# Build locales to avoid encoding issues with Scrapy encoding
```

Comment:
this is Dockerfile-periodic. i wonder whether the other Dockerfile needs changing too?

Reply:
At the moment we're not really using any other dockerfile. I should probably just get rid of them 🤔